cloudformation/cmsfronts.yml

```diff
@@ -67,6 +67,21 @@ Resources:
                                     - Ref: "AWS::AccountId"
                                     - ":table/"
                                     - Ref: DynamoDBTable
+                    -   Effect: Allow
+                        Action:
+                            - "dynamodb:PutItem"
+                            - "dynamodb:UpdateItem"
+                            - "dynamodb:GetItem"
+                        Resource:
+                            - Fn::Join:
```

Comment:
Could this be done with `!Sub` instead of join?

Reply:
I don't thin so because I need to be able to reference the dynamotable to but together the table arn. I couldn't figure out how to do it using !Sub.